benchmark/scripts/generate_harness/generate_harness.py

```diff
@@ -12,43 +12,70 @@
 #
 # ===---------------------------------------------------------------------===//
 
-# Generate CMakeLists.txt and utils/main.swift from templates.
+# Generate boilerplate, CMakeLists.txt and utils/main.swift from templates.
 
 from __future__ import print_function
 
-import glob
+import argparse
+import jinja2
```

Comment:
GYB is clearly superior for code templating due to its use in `stdlib` - hence my motivation for this PR. But I don’t see a way to use it to generate the `CMakeLists.txt`: `generate_harness.py` walks directories scans files and feeds the results into jinja2 template. I don’t see the way to externally parametrize `gyb` invocations. `.swift.gyb` files are AFAIK self-contained.

Are we now debating merits of my PR or general state of prior technology use in the project?